src/rules/no-array-length-in-deps.ts

```
@@ -0,0 +1,276 @@
+	let candidate = `${base}Hash`;
+	if (!taken.has(candidate)) return candidate;
+	let i = 2;
+	while (taken.has(`${candidate}${i}`)) {
+		i++;
+	}
+	return `${candidate}${i}`;
+}
+
+function collectAllTakenNames(context: TSESLint.RuleContext<MessageIds, []>): Set<string> {
+	const names = new Set<string>();
+	let scope: any = context.getScope();
```

ESLint

[error] Replace `↹` with `··`

(prettier/prettier)